### PR TITLE
Ruby: Word Count mentoring whitespace and newlines

### PR DESCRIPTION
**Pull request by kotp**
Brought to attention by @alkhatib #1851

### CODE REVIEW

**Review by alkhatib-shopify — DISMISSED** [review]

**Review by SleeplessByte — APPROVED** [review]
Thanks both!

**Comment by SleeplessByte** [comment]
Let's wait for #1854 before merging this?

**Comment by kotp** [comment]
#1854 is in…

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Great stuff!

File: tracks/ruby/exercises/word-count/mentoring.md [hunk 1]
```diff
@@ -22,39 +24,39 @@ class WordCount
...
+- The RegEx gets complicated in the last test, where it should catch both `"large"` and `'large'` `=> { large: 2 }`.
+As long as they got something that catches a word with or without `'`, it's fine to give away the last step for free.
+http://www.rubular.com/
 
 ### Talking points
+
 - `inject`/`reduce`: see note under Reasonable Variants.
 - Extracting the preparation of the input into a separate (private) method.
-- `intermediate steps to strip quotes`: One strategy people use to catch quoted words is an intermediate step like 'strip_quotes'. Suggest to solve it within the RegEx. 
+- `intermediate steps to strip quotes`: One strategy people use to catch quoted words is an intermediate step like 'strip_quotes'. Suggest to solve it within the RegEx.
```
Comment (kotp) [inline tracks/ruby/exercises/word-count/mentoring.md:54]: I missed it, but this should not have backticks, the formatting is wrong.  It looks like "code" when it should not.

Comment (SleeplessByte) [reply tracks/ruby/exercises/word-count/mentoring.md:54]: I missed it too. Thought it was normal quotes

**Pull request merged**